dc.sh

```diff
@@ -43,6 +44,11 @@ function up {
    "jealous bright oyster fluid guide talent crystal minor modify broken stove spoon pen thank action smart enemy chunk ladder soon focus recall elite pulp"
 
   docker ps -a
+
+    if [ "$SETUP_SMART_CONTRACT" = true ]; then
```

Comment:
so originally i was passing true in such as `./dc.sh -c true bounce` i've since changed this to not need to accept an argument. `./dc.sh -c bounce` works as expected